packages/network/src/helpers/trackerHttpEndpoints.ts

```diff
@@ -99,6 +100,21 @@ export function trackerHttpEndpoints(
             return addRttsToNodeConnections(nodeId, Array.from(neighbors), tracker.getOverlayConnectionRtts())
         }))
     })
+    app.get('/nodes/:nodeId/streams', async (req: express.Request, res: express.Response) => {
+        const nodeId = req.params.nodeId
+        staticLogger.debug(`request to /nodes/${nodeId}/streams`)
+        const result = Object.entries(tracker.getOverlayPerStream())
```

Comment:
Should this be in logic/trackerSummaryUtils.ts ?

Reply:
Good point, I'll move it there